### DISCUSSION TITLE
Getting a 404 for CSS/JS assets

### DISCUSSION THREAD

**Discussion by BlipRanger** [gollum/gollum#1864] [Q&A]
Hello, I just spun up a quick docker instance of Gollum and when I browse to it it fails to load basically all CSS/JS assets. I notice on the server side I'm getting errors like this:

`GET /gollum/assets/app-309be032396e783b13a47df58f389b7c8e11c2b2d42640560b874f677c25f6e5.css HTTP/1.1" 404 90
`
`GET /gollum/assets/print-512498c368be0d3fb1ba105dfa84289ae48380ec9fcbef948bd4e23b0b095bfb.css HTTP/1.1" 404 92`
Did I miss something in setup? 
![image](https://user-images.githubusercontent.com/1860540/199519699-ead246e0-3577-4342-b9f1-3dd183f8248b.png)

**Comment by dometto** [comment]
Hi @BlipRanger! Can you let us know how you're running gollum exactly? Is it possible that you're running an older version of the Docker image?

I can't reproduce the issue with:

```
docker pull gollumwiki/gollum:master
docker run --rm -p 4567:4567 -v /tmp/lotr:/wiki gollumwiki/gollum:master
```

**Reply by BlipRanger** [reply]
> docker pull gollumwiki/gollum:master
> docker run --rm -p 4567:4567 -v /tmp/lotr:/wiki gollumwiki/gollum:master

Just did exactly that; I've also tried the Docker-Compose setup as well (per the wiki here). Does there need to be something in the wiki volume (besides an initialized git repo)?

**Reply by BlipRanger** [reply]
Additional info:
Host OS: Debian 10
Docker Version: 18.09
I am accessing the server from a browser on a different machine, so ip of the server plus port 4567.

**Reply by BlipRanger** [reply]
Weirder still, I went ahead and deployed it to my prod server (which has docker 20.10, uses compose) and it worked just fine. 
I might try updating my dev server's docker version and see if it makes a difference.

**Reply by dometto** [reply]
Glad you got it running! I assume you are running different versions of the image on both systems? If so, I assume `docker pull` did not update the image to the latest version on your first system. I am using `8b680add568f` locally.
